src/main/java/Lesson8/IntegerList.java

```diff
@@ -0,0 +1,85 @@
+package Lesson8;
+
+import java.util.Arrays;
+
+
+public class IntegerList implements OwnList {
+
+    private int size = 0;
+
+    public  Integer[] arrayAsList;
+
+    public IntegerList() {
+        arrayAsList = new Integer[10];
+    }
+
+    @Override
+    public int size() {
+        return size;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        if (arrayAsList.length == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public void add(Integer element) {
+        if (arrayAsList.length <= 1) {
```

Comment:
Nie rozumiem tego. Czy to oznacza, że tylko podczas pierwszego "add" (zakładając, że arrayAsList ma length 0 lub 1) zwiększysz tę tablicę? A co jeżeli wywołam add(coś) milion razy?